.github/workflows/loadtest.yaml

```diff
@@ -0,0 +1,40 @@
+name: CI Stage Load Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      users:
+        type: string
+        description: Number of Users
+      threads:
+        type: string
+        description: Number of threads
+
+jobs:
+  load_test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up jq
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y jq
+
+    - name: Prepare Load Test
+      working-directory: ./tests/load-tests
+      env:
+        STAGING_USERS_1: ${{ secrets.STAGING_USERS_1 }}
+        STAGING_USERS_2: ${{ secrets.STAGING_USERS_2 }}
+        STAGING_USERS_3: ${{ secrets.STAGING_USERS_3 }}
+        STAGING_USERS_4: ${{ secrets.STAGING_USERS_4 }}
+      run: ./ci-scripts/merge-json.sh $STAGING_USERS_1 $STAGING_USERS_2 $STAGING_USERS_3 $STAGING_USERS_4
```

Comment:
I assume there is some secrets size limit. Looks like we could have that file GPG encrypted: https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-large-secrets. Or maybe get it from BitWarden https://bitwarden.com/help/github-actions-integration/ or Vault https://github.com/marketplace/actions/vault-secrets ?

Reply:
i have split that file into 4 and encoded it in base64, and i am decoding it in this step